modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/utils/index.js

```diff
@@ -97,11 +97,20 @@ export function getValueDetailsFromItem(item, fieldName) {
 		return null;
 	}
 
+	const i18nRawTextFieldName = `${fieldName}RawText`;
+
 	let rootPropertyName = fieldName;
 	const valuePath = [];
 	let navigatedValue = item;
 
-	if (Array.isArray(fieldName)) {
+	if (
+		Object.hasOwn(item, i18nRawTextFieldName) &&
+		Object.values(i18nRawTextFieldName).length
+	) {
+		valuePath.push(fieldName);
+		navigatedValue = navigatedValue[i18nRawTextFieldName];
+	}
+	else if (Array.isArray(fieldName)) {
```

Comment:
Is there a need for `*RawText` property check? I might be good enough just to check if item property name ends with `_i18n`. Maybe, for extra safety, we can check if Item property value is an object. A customer could theoretically create a field named `*_i18n`, but they could also create a field named `*RawText`.

Unrelated to your PR, naming in this file is terrible :disappointed: `valuePath`, `navigatedValue`, `formattedProperty` etc, all so difficult to understand.

Reply:
API's with fields of type `richText` have an extra field in the response where they store the selected translated value after being processed to strip out all HTML tags from the string. 

Anyway, I'm trying to find a way to remove this logic from the FDS component as it is only related with the Dataset manager.